modules/messages/tests/server/message-to-influx.server.service.tests.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+    path = require('path'),
+    async = require('async'),
+    User = mongoose.model('User'),
+    Message = mongoose.model('Message'),
+    messageToInfluxService =
+  require(path.resolve('./modules/messages/server/services/message-to-influx.server.service'));
+require('should');
+
+/**
+ * Unit tests
+ */
+describe('Message to influx server service Unit Tests:', function() {
+  var user1,
+      user2;
+
+  // here we create the users
+  beforeEach(function(done) {
+
+    user1 = new User({
+      firstName: 'Full',
+      lastName: 'Name',
+      displayName: 'Full Name',
+      email: 'user1@test.com',
+      username: 'username1',
+      password: 'password123',
+      provider: 'local'
+    });
+
+    user2 = new User({
+      firstName: 'Full',
+      lastName: 'Name',
+      displayName: 'Full Name',
+      email: 'user2@test.com',
+      username: 'username2',
+      password: 'password123',
+      provider: 'local'
+    });
+
+    // save those users to mongoDB
+    user1.save(function(err) {
+      if (err) done(err);
+      user2.save(function(err) {
+        if (err) done(err);
+        done();
+      });
+    });
+  });
+
+  // after each test removing all the messages and users
+  afterEach(function(done) {
+    Message.remove().exec(function() {
+      User.remove().exec(done);
+    });
+  });
+
+  describe('Testing messageToInfluxService.process', function () {
+
+    // defining some messages which will be later used for testing
+    // note: the userFrom and userTo is not existent at this point, yet
+    var message1to2 = { content: 'message content' };
+    var message2to1 = { content: 'message content' };
+    var shortMessage = { content: 'short content' };
+    // setting a long message
+    var longString = '0123456789';
+    for (var i = 0; i < 5; ++i) {
```

Comment:
Yep! This should do the trick:
```javascript
_.repeat('*', 3);
// => '***'
```
Might be we don't need the complexity of modifying config + rerunning right now.